File: mycollab-services/src/main/java/com/mycollab/module/crm/domain/criteria/TargetGroupSearchCriteria.java
package com.mycollab.module.crm.domain.criteria;

import com.mycollab.db.arguments.NumberSearchField;
import com.mycollab.db.arguments.SearchCriteria;
import com.mycollab.db.arguments.StringSearchField;

public class TargetGroupSearchCriteria extends SearchCriteria {
    private StringSearchField assignUserName;

    private StringSearchField listName;

    private StringSearchField assignUser;

    private NumberSearchField targetId;

    private NumberSearchField contactId;

    private NumberSearchField leadId;
    
    private NumberSearchField campaignId;

	public StringSearchField getAssignUserName() {
		return assignUserName;
	}

	public void setAssignUserName(StringSearchField assignUserName) {
		this.assignUserName = assignUserName;
	}

	public StringSearchField getListName() {
		return listName;
	}

	public void setListName(StringSearchField listName) {
		this.listName = listName;
	}

	public StringSearchField getAssignUser() {
		return assignUser;
	}

	public void setAssignUser(StringSearchField assignUser) {
		this.assignUser = assignUser;
	}

	public NumberSearchField getTargetId() {
		return targetId;
	}

	public void setTargetId(NumberSearchField targetId) {
		this.targetId = targetId;
	}

	public NumberSearchField getContactId() {
		return contactId;
	}

	public void setContactId(NumberSearchField contactId) {
		this.contactId = contactId;
	}

	public NumberSearchField getLeadId() {
		return leadId;
	}

	public void setLeadId(NumberSearchField leadId) {
		this.leadId = leadId;
	}

	public NumberSearchField getCampaignId() {
		return campaignId;
	}

	public void setCampaignId(NumberSearchField campaignId) {
		this.campaignId = campaignId;
	}
    
}

File: mycollab-services/src/main/java/com/mycollab/module/crm/domain/criteria/TargetGroupSearchCriteria.kt
/**
 * Copyright © MyCollab
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU Affero General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU Affero General Public License for more details.
 *
 * You should have received a copy of the GNU Affero General Public License
 * along with this program.  If not, see <http:></http:>//www.gnu.org/licenses/>.
 */
package com.mycollab.module.crm.domain.criteria

import com.mycollab.db.arguments.NumberSearchField
import com.mycollab.db.arguments.SearchCriteria
import com.mycollab.db.arguments.StringSearchField

class TargetGroupSearchCriteria : SearchCriteria() {
    var assignUserName: StringSearchField? = null

    var listName: StringSearchField? = null

    var assignUser: StringSearchField? = null

    var targetId: NumberSearchField? = null

    var contactId: NumberSearchField? = null

    var leadId: NumberSearchField? = null

    var campaignId: NumberSearchField? = null

}
